pkg/user/manager/ldap/ldap.go

```diff
@@ -97,39 +101,51 @@ func New(m map[string]interface{}) (user.Manager, error) {
 		return nil, err
 	}
 
-	return &manager{
-		hostname:     c.Hostname,
-		port:         c.Port,
-		baseDN:       c.BaseDN,
-		userfilter:   c.UserFilter,
-		groupfilter:  c.GroupFilter,
-		bindUsername: c.BindUsername,
-		bindPassword: c.BindPassword,
-		idp:          c.Idp,
-		schema:       c.Schema,
-	}, nil
+	// backwards compatibility
+	c.UserFilter = strings.ReplaceAll(c.UserFilter, "%s", "{{.OpaqueId}}")
+	if c.FindFilter == "" {
+		c.FindFilter = c.UserFilter
+	}
+	c.GroupFilter = strings.ReplaceAll(c.GroupFilter, "%s", "{{.OpaqueId}}")
```

Comment:
Fallback to UserFilter if it's empty?

Reply:
The group filter is completely broken imo. This pr just replaces the `%s` in case the config wes not updated. I will look at group related stuff next week.

Reply:
Sure!